gpu/gpu.go

```diff
@@ -173,6 +175,34 @@ func GetCPUInfo() GpuInfoList {
 	return GpuInfoList{cpus[0].GpuInfo}
 }
 
+func DetectInteliGpuMemStatus(gpuInfo *OneapiGPUInfo) {
+	var totalram uint64 = uint64(C.check_total_host_mem())
+	// there will be half of total ram can be handle as iGPU vram
+	gpuInfo.TotalMemory = totalram / 2
+	cmd := "sed 's/^.*, \\([0-9]\\+\\) bytes$/\\1/' /sys/kernel/debug/dri/0/i915_gem_objects | awk '{print $1; exit}'"
```

Comment:
It appears that iGPU has sometimes index `1`, not always have index `0`, even if host does not have any other GPUs.

Therefore it may be better just take first line from first file with lowest number:
```suggestion
	cmd := "head -q -n 1 /sys/kernel/debug/dri/[0-9]/i915_gem_objects | head -n 1 | awk '{n=NF-1; print $n}'"
```

Note: while `i915_gem_objects` file first line content has changed over the years, total amount of used memory has always been in the second last field (`n=FN-1`).  If relying on that is not enough, awk in above could replaced with (suitable quoted) sed: `sed -e 's/^.* \([0-9]\+\) bytes$/\1/'`

Reply:
looks like `head -q -n 1 /sys/kernel/debug/dri/[0-9]/i915_gem_objects | head -n 1 | awk '{n=NF-1; print $n}'` can't work on my machine, use `head -q -n 1 $(find /sys/kernel/debug/dri -maxdepth 1 -type d -name '[0-9]*' | sort | head -n 1)/i915_gem_objects | awk '{n=NF-1; print $n}'` instead.

Reply:
Odd, that shell patterns do not work.

If you're using `find`, one option would be just taking largest system RAM GPU usage, on assumption that it's the iGPU:
`find /sys/kernel/debug/dri/ -name i915_gem_objects | xargs awk '{n=NF-1; print $n; exit}' | sort -nr | head -1`

Reply:
accept, Thanks!